stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1771,6 +1892,20 @@ func _TFCOpSetAttrString(_ op: CTFEOp,
   }
 }
 
+/// Wrapper around TFE_OpSetAttrString that handles converting the Swift Stdlib
+/// String into a buffer that TFE_OpSetAttrString can read.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpSetAttrFunctionName")
```

Comment:
To interoperate with C, use `@_cdecl`.
```suggestion
@_cdecl("_swift_tfc_OpSetAttrFunctionName")
```

Reply:
This is being used in irgen.

Reply:
In that case this should be cdecl. Silgen name does not give you calling convention guarantees.

Reply:
As far as I understand, we are simply using this annotation to make it create calls to these functions in IRGen. See other functions defined here with `_silgen_name` and [`RuntimeFunctions.def`](https://github.com/apple/swift/blob/471137b0734e19bab97a92108202b3b2431ff209/include/swift/Runtime/RuntimeFunctions.def)